DMCompiler/DMStandard/_Standard.dm

```diff
@@ -14,6 +14,7 @@ proc/ckey(Key)
 proc/ckeyEx(Text)
 proc/clamp(Value, Low, High)
 proc/cmptext(T1)
+proc/cmptextEx(T1)
```

Comment:
These should have their `opendream_unimplemented` attribute set so using them leaves a warning
```suggestion
proc/cmptextEx(T1)
	set opendream_unimplemented = TRUE
```